enterprise/spring.webmvc/src/org/netbeans/modules/spring/webmvc/resources/spring-webmvc-5.0.xml

```diff
@@ -25,7 +25,7 @@
     <localizing-bundle>org.netbeans.modules.spring.webmvc.resources.Bundle</localizing-bundle>
     <volume>
         <type>classpath</type>
-        <resource>jar:nbinst://org.netbeans.libs.springframework/modules/ext/spring-5/spring-webmvc-5.2.2.RELEASE.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.libs.springframework/modules/ext/spring-5/spring-webmvc-5.2.9.RELEASE.jar!/</resource>
```

Comment:
`Srpingmvc` module actually delivers 5.2.9 version; I've updated. @hectorespert to review.

Reply:
Please, rebase the last changes from master, I just merged the fix for that